frontend/src/components/v1/StatusPill.css

```
@@ -0,0 +1,86 @@
+.status-pill {
+  --status-pill-bg: color-mix(in oklab, var(--surface-elevated, rgba(255, 255, 255, 0.08)) 92%, transparent);
+  --status-pill-border: color-mix(in oklab, var(--glass-border, rgba(255, 255, 255, 0.16)) 86%, transparent);
+  --status-pill-text: var(--text-main, #f5f7fb);
+  --status-pill-icon: currentColor;
+  display: inline-flex;
```

Stylelint (17.7.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)